src/modules/deal-folder/dto/upload-file-in-deal-folder-request.dto.ts

```diff
@@ -33,9 +33,10 @@ export class UploadFileInDealFolderRequestItem {
 
   @ValidatedStringApiProperty({
     description: 'The path of the location/folder in Azure storage from which the file needs to be loaded.',
-    minLength: 1,
+    minLength: 24,
     maxLength: 250,
-    pattern: /^[\w\-:/\\()\s]+$/,
+    pattern: /^[a-fA-F\d]{24}(\/[\w\-:/\\()\s]*)*$/,
```

Comment:
Unfortunately the regex string `/^[a-fA-F\d]{24}(\/[\w\-:/\\()\s]*)*$/` is vulnerable to ReDOS attack according to https://devina.io/redos-checker

Reply:
Changed to `/^[a-fA-F\d]{24}(\/[\w\-:\\()\s]+)*$/`, which is not vulnerable and also ensures the string does not end with '/' (which causes a 500 error)